TI_Practice4/Pactice4.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+
+namespace TI_Practice4
+{
+    class Practice4
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Please write something.");
+            String input = Console.ReadLine();
```

Comment:
please, user string instead of String since you don't need any String object properties/methods here